client/workflows/client.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/mendersoftware/go-lib-micro/identity"
 	"github.com/mendersoftware/go-lib-micro/requestid"
 	"github.com/mendersoftware/go-lib-micro/rest_utils"
+	"github.com/mendersoftware/inventory/utils"
 	"github.com/pkg/errors"
```

Comment:
minor: I am unsure how we sort imports nowadays, I thought we are doing it like that:
```go
	"bytes"
	"context"
	"encoding/json"
	"net/http"
	"strings"
	"time"

	"github.com/pkg/errors"

	"github.com/mendersoftware/go-lib-micro/identity"
	"github.com/mendersoftware/go-lib-micro/requestid"
	"github.com/mendersoftware/go-lib-micro/rest_utils"

	"github.com/mendersoftware/inventory/utils"
```
but I can see that most of the inventory service does follow a different pattern.

Reply:
Yeah, imports are not sorted like that all across the repo. The problem is that touching them is going to generate tons of conflicts with the ENT. Not sure it's a good idea...

Reply:
let's forget about it, it is minor.